README.md

```diff
@@ -16,7 +16,7 @@ This game is a strategy board game in which two to four players act as the leade
 If you want to know more about the rules and the documentation of the game you can click [here](https://niskut.github.io/Civilization/index.html).
```

Comment:
Error on the branch and PR name ! Improvement-22 already exist. Please be sure you change the name **BEFORE** merging!